### ISSUE TITLE
Cannot export private key from the keybase website 

### ISSUE THREAD

**Issue by bartBackbase** [keybase/client#24934]
Connected with log sent directly from the console:
I cannot export my key from the keybase website.

<img width="725" alt="image" src="https://user-images.githubusercontent.com/98871438/164972344-8f15e44c-7170-448d-a2d0-52ca43d16743.png">

After entering proper pass-phrase i get following error: 

<img width="610" alt="image" src="https://user-images.githubusercontent.com/98871438/164972321-ca09b5c7-539d-413f-8f8a-6f5ac81eb9a2.png">

my log id: ac15698bfafe61f18ecbf81c

**Comment by CitizenPrayer** [comment]
I Concur, I get the exact same issue. I've tried troubleshooting this every way possible, and I am unable to export my keybase private key.

**Comment by CitizenPrayer** [comment]
I agree, this is an issue I ran into also. I'm getting the same exact error when I try to export my key. I've even gone so far as to reset my passphrase and made certain that the passphrase was correct. I've done extensive troubleshooting, and so far this does not work from multiple devices, browsers, and operating systems, as well as apps. Quite simply, we are unable to export our private keys from the website.

**Comment by zapu** [comment]
This should be fixed now, can you try that again?
